pkgs/games/cataclysm-dda/bn-git.nix

```
@@ -0,0 +1,63 @@
+{ lib, callPackage, fetchFromGitHub, fetchpatch, pkgs, wrapCDDA, attachPkgs
+, tiles ? true
+, debug ? false
+, useXdgDir ? false
+, version ? "v0.8.0-unstable-2025-05-17"
+, rev ? "4192d73997fa78a05e06b99fae8e34dd021226c5"
+, sha256 ? "sha256-jeyyJw5tkgvVgvpnDj5XkobzP8NBMd0YFk2ShTRcd9U="
+}:
+
+let
+  desktopFilePath = "";
+
+  common = callPackage ./common.nix {
+    inherit tiles debug useXdgDir desktopFilePath;
+  };
+
+  self = common.overrideAttrs (common: rec {
+    pname = "cataclysm-bn-git";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "cataclysmbnteam";
+      repo = "Cataclysm-BN";
+      inherit rev sha256;
+    };
+
+    patches = [
+      # Unfortunately some READMEs in data are symlinks to doc which is not included
+      # resulting in dangling symlinks breaking build
+      ./delete-dangling-readme.patch
+    ];
```

It would probably be simpler to get rid of the patch file, and just do something like

```suggestion
    postInstall = ''
      # Unfortunately some READMEs in data are symlinks to doc which is not included
      # resulting in dangling symlinks breaking build
      rm $out/share/cataclysm-bn/json/{external_tileset/README.md,mapgen/lab/README.md}
    '';
```

(I'm not sure if this is the exact path, but this is the idea)